landing-pages/site/content/en/blog/airflow-2.7.0/index.md

```diff
@@ -0,0 +1,84 @@
+---
+title: "Apache Airflow 2.7.0 is here"
+linkTitle: "Apache Airflow 2.7.0 is here"
+author: "Jed Cunningham"
+github: "jedcunningham"
+linkedin: "jedidiah-cunningham"
+description: "Apache Airflow 2.7.0 has been released!"
+tags: [Release]
+date: "2023-08-18"
+---
+
+I’m happy to announce that Apache Airflow 2.7.0 has been released! Some notable features have been added that we are excited for the community to use.
+
+Apache Airflow 2.7.0 contains over 500 commits, which include 40 new features, 49 improvements, 53 bug fixes, and 15 documentation changes.
+
+**Details**:
+
+📦 PyPI: https://pypi.org/project/apache-airflow/2.7.0/ \
+📚 Docs: https://airflow.apache.org/docs/apache-airflow/2.7.0/ \
+🛠 Release Notes: https://airflow.apache.org/docs/apache-airflow/2.7.0/release_notes.html \
+🐳 Docker Image: "docker pull apache/airflow:2.7.0" \
+🚏 Constraints: https://github.com/apache/airflow/tree/constraints-2.7.0
+
+## Setup and Teardown (AIP-52)
+
+Airflow now has first class support for the concept of setup and teardown tasks. These tasks have special behavior in that:
+
+* Teardown tasks will still run, no matter what state the upstream tasks end up in
+* Teardown tasks failing won’t, by default, cause the DAG run to fail
+* Automatically clear setup/teardown tasks when clearing a dependent task
+
+You can read more about setup and teardown in the [Introducing Setup and Teardown tasks blog post]({{< ref "blog/introducing_setup_teardown/index.md" >}}), or in the [setup and teardown docs](https://airflow.apache.org/docs/apache-airflow/2.7.0/howto/setup-and-teardown.html).
+
+## Cluster Activity UI
+
+There is a new top level page in Airflow, the Cluster Activity page. This gives an overview of the cluster, including component health, dag and task state counts, and more!
+
+![New cluster activity page](cluster_activity.png)
+
+## Graph and gantt views moved into the Grid view UI
+
+The graph and gantt views have been rewritten and moved into the now familiar grid view. This makes it easier to jump between task details, logs, graph, and gantt views without losing your place in a complicated DAG.
+
+![Graph in grid view](graph_in_grid.png)
+
+## Enable deferrable mode for all deferable tasks with 1 config setting
+
+Airflow 2.7.0 comes with a new config option, `default_deferrable`, which allows admins to enable deferrable mode for all deferrable tasks without requiring any DAG modifications. Simply set it in your config and enjoy async tasks!
+
+## OpenLineage provider
```

Comment:
Lets rename it to OpenLineage built-in integration?
The provider is part of this integration and i'd like to avoid confusion as the provider itself is not part of the core release

Reply:
Agree.